packages/appstore/src/components/onboarding/static-dashboard.tsx

```diff
@@ -0,0 +1,423 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Text, ButtonToggle, Icon } from '@deriv/components';
+import { isMobile, isDesktop } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import StaticCFDAccountManager from './static-cfd-account-manager';
+import StaticPlatformLauncher from './static-platform-launcher';
+import StaticAppLauncher from './static-applauncher';
+
+import './static-dashboard.scss';
+
+type TStaticDashboard = {
+    loginid?: string;
+    is_grey?: boolean;
+    currency?: string;
+    has_account?: boolean;
+    is_last_step?: boolean;
+    derived_amount?: string;
+    is_get_blurry?: boolean;
+    financial_amount?: string;
+    is_icon_blurry?: boolean;
+    is_trade_blurry?: boolean;
+    is_topup_blurry?: boolean;
+    is_text_animated?: boolean;
+    is_trade_animated?: boolean;
+    is_topup_animated?: boolean;
+    is_button_animated?: boolean;
+    is_cfd_item_blurry?: boolean;
+    is_cfd_text_blurry?: boolean;
+    is_options_item_blurry?: boolean;
+    is_options_text_blurry?: boolean;
+    is_derivx_last_step?: boolean;
+    is_financial_last_step?: boolean;
+    has_applauncher_account?: boolean;
+    is_cfd_description_blurry?: boolean;
+    is_platformlauncher_blurry?: boolean;
+    is_options_description_blurry?: boolean;
+};
+
+const StaticDashboard = ({
+    loginid,
+    is_grey,
+    currency,
+    has_account,
+    is_last_step,
+    is_get_blurry,
+    is_icon_blurry,
+    is_topup_blurry,
+    is_trade_blurry,
+    financial_amount,
+    is_text_animated,
+    is_trade_animated,
+    is_topup_animated,
+    is_button_animated,
+    is_cfd_item_blurry,
+    is_cfd_text_blurry,
+    is_derivx_last_step,
+    is_financial_last_step,
+    is_options_item_blurry,
+    is_options_text_blurry,
+    has_applauncher_account,
+    is_cfd_description_blurry,
+    is_platformlauncher_blurry,
+    is_options_description_blurry,
+}: TStaticDashboard) => {
+    const Divider = () => <div className='divider' />;
+
+    const toggle_options = [
+        { text: 'CFDs', value: 'CFDs' },
+        { text: 'Options', value: 'Options' },
+    ];
+
+    const [index, setIndex] = React.useState(0);
+
+    React.useEffect(() => {
+        const changeIndex = setInterval(() => {
+            if (index === 0) {
+                setIndex(1);
+            } else {
+                setIndex(0);
+            }
+        }, 5000);
+
+        return () => clearInterval(changeIndex);
+    }, [index]);
```

Comment:
```suggestion
    }, [index]);
```
Wouldn't this cause multiple setIntervals to run if the `index` were to change too many time?